src/components/project/dto/list-projects.dto.ts

```diff
@@ -65,9 +65,9 @@ export abstract class ProjectFilters {
   @Field({
     nullable: true,
     description:
-      'When a project works on more than one language it is a "cluster" project. It has more than one engagement',
+      'When a project has more than one engagement.  Cluster = Traslation Projects and Cohort = Internship Projects',
   })
-  readonly clusters?: boolean;
+  readonly clusterOrCohort?: boolean;
```

Comment:
This is the only place these nouns are used in the API, maybe something like `onlyMultipleEngagements` would be better...

Reply:
agree, added